### PR TITLE
Add Index file

### PR DESCRIPTION
**Pull request by sameedmohsin28**
I created this project by following the instructions given in the task description. I was unable to do it on my own and my learning partner was also unavailable, so I sought help from my morning session and standup session partners. I faced major difficulties in installing node.js and setting up linters.yml file.

### CODE REVIEW

**Review by joskalenda — CHANGES_REQUESTED** [review]
Hi @sameedmohsin28 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌

> You have submitted a project, that no pull request summary and failed linters, kindly update these and resubmit your work



Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

### Cheers and Happy coding!👏👏👏

------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

-----
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._ 🟢

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
-# Hello-World
\ No newline at end of file
+# Hello-World
+You do not need any pre requisites nor any instructions for this project. This is a basic project to get us started with html, css, git, GitHub, and linters.
```
Comment (zieeco) [inline README.md]: - Hey, great job with this project so far, I admire your energy, however, the purpose of this project is to help you get familiar with setting up a professional GitHub repository and get comfortable in doing them repetitively
   - Let's improve our readme by using this [template](https://github.com/microverseinc/readme-template) but you should customize it to suit this current project's needs. 
   - Replace all the placeholder text as they're there meant to guide on how to add some content in the readme
   > Follow the template closely to update them with your personal content but maintain its structure and format.

File: test.md [hunk 2]
```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```
Comment (zieeco) [inline test.md]: - As this is clearly stated that `I should be ignored by git`, kindly add this file in the `.gitignore` file.

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1 @@
+node_modules/
```
Comment (zieeco) [inline .gitignore]: - Add `test.md` so it would be ignored by git

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @sameedmohsin28 ,


Great job with the `Hello Microverse` project! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice project setup. ✔️
- Fair PR description. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 4]
```diff
@@ -1 +1,20 @@
...
+GitHub: @sameedmohsin28
+LinkedIn: https://www.linkedin.com/in/sameed-mohsin-538792180/ 
+
+
+If you like this project, please send an appreciation on my LinkedIn
+
+
+🙏 Acknowledgments
+
+I would like to thank https://github.com/matee-safi, https://github.com/mpa-mxiang, https://github.com/baezluise for their support.
```
Comment (abdulhamiid) [inline README.md]: As requested by the previous reviewer and [project requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) please kindly open the link [MICROVERSE readme template](https://github.com/microverseinc/readme-template) and customize it to your project ☺

Kindly replace all the placeholder text as they're there meant to guide on how to add some content in the readme 👍

File: .gitignore [hunk 5]
```diff
@@ -0,0 +1,2 @@
...
+test.md
```
Comment (abdulhamiid) [inline .gitignore]: The purpose of gitignore files is to ensure that specific files not tracked by Git remain untracked, which means that test.md file should not be present on your GitHub repository please kindly remove it.

👇 
image

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @sameedmohsin28, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, I really appreciate the effort you put in! but there are still some which haven't been addressed yet.

almost-there

## Highlights 🌟

- ✔️ PR descriptions are descriptive
- ✔️ Linters checks are passing
- ✔️ Use GitHub Flow
- ✔️ Meaningful commit messages.

## Required Changes ♻️
_Please kindly check the comments under the review._

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by abdulhamiid** [comment]
At first, you will have to remove the `test.md` file manually from your GitHub repository

So subsequent time, when you push any changes to GitHub, `.gitignore` file will ensure that `test.md` file is ignored and not pushed to the remote repository.

**Review by NduatiKagiri — APPROVED** [review]
Hi @sameedmohsin28 ,

# APPROVED 💯  

Good job making the requested changes. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

ea0cb5a48e3337523ffe8d60f177879d

## Highlights ✔️ 
✅ GitHub Flow observed
✅ No Linters errors

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**